app/src/main/kotlin/com/simplemobiletools/flashlight/helpers/LollipopCameraFlash.kt

```diff
@@ -0,0 +1,37 @@
+@file:Suppress("DEPRECATION")
+
+package com.simplemobiletools.flashlight.helpers
+
+import android.graphics.SurfaceTexture
+import android.hardware.Camera
+
+class LollipopCameraFlash : CameraFlash {
```

Comment:
I couldn't test this on Android 5. I ensured that it is consistent with what was on the `MyCameraImpl` class